content/en/courses/350/blog-project/admin.md

```diff
@@ -23,37 +23,36 @@ draft: False
 Switch to branch **3-admin** from **malmarz/isom350-blog** github repo to see this step's implementation
 {{% /callout %}}
 
-The admin interface allows the website administrator to manage the data for the web application. However, before you can use it you must tell Django which data models you want managed and you must setup the admin interface.
+لتسمح لواجهه الإدارة لمسؤول عن الموقع بإدارة البيانات لتطبيق الويب. ومع ذلك ، قبل أن تتمكن من استخدامه ، يجب أن تخبر Django بنماذج البيانات التي تريد إدارتها ويجب عليك مسؤل واجهة الإدارة.
 
 First, check the file mysite/urls.py, and check whether the following line exists:
```

Comment:
translate all english here and remove english from this file (except inside the code_